PreFreeSurfer/PreFreeSurferPipeline.sh

```diff
@@ -415,354 +503,491 @@ fi
-  #     refinement to the "T2wtoT1w" registration that FreeSurfer provides.
-
-  # Just implement inline, rather than writing a separate script
-  # Added 2/19/2019
-  # ------------------------------------------------------------------------------
-
-  log_Msg "Creating one-step resampled version of {T1w,T2w}_acpc_dc outputs"
-
-  # T1w
-  OutputOrigT1wToT1w=OrigT1w2T1w_PreFS  # Name for one-step resample warpfield
-  convertwarp --relout --rel --ref=${T1wTemplate} --premat=${T1wFolder}/xfms/acpc.mat --warp1=${T1wFolder}/xfms/${T1wImage}_dc --out=${T1wFolder}/xfms/${OutputOrigT1wToT1w}
-
-  OutputT1wImage=${T1wFolder}/${T1wImage}_acpc_dc
-  applywarp --rel --interp=spline -i ${T1wFolder}/${T1wImage} -r ${T1wTemplate} -w ${T1wFolder}/xfms/${OutputOrigT1wToT1w} -o ${OutputT1wImage}
-  fslmaths ${OutputT1wImage} -abs ${OutputT1wImage} -odt float  # Use -abs (rather than '-thr 0') to avoid introducing zeros
-  fslmaths ${OutputT1wImage} -div ${T1wFolder}/BiasField_acpc_dc ${OutputT1wImage}_restore
-  fslmaths ${OutputT1wImage}_restore -mas ${T1wFolder}/${T1wImage}_acpc_dc_brain ${OutputT1wImage}_restore_brain
-
-  if [ ! "${T2wInputImages}" = "NONE" ] ; then
-    OutputOrigT2wToT1w=OrigT2w2T1w_PreFS  # Name for one-step resample warpfield
-    convertwarp --relout --rel --ref=${T1wTemplate} --premat=${T2wFolder}/xfms/acpc.mat --warp1=${T1wFolder}/xfms/${T2wImage}_reg_dc --out=${T1wFolder}/xfms/${OutputOrigT2wToT1w}
-
-    OutputT2wImage=${T1wFolder}/${T2wImage}_acpc_dc
-    applywarp --rel --interp=spline -i ${T2wFolder}/${T2wImage} -r ${T1wTemplate} -w ${T1wFolder}/xfms/${OutputOrigT2wToT1w} -o ${OutputT2wImage}
-    fslmaths ${OutputT2wImage} -abs ${OutputT2wImage} -odt float  # Use -abs (rather than '-thr 0') to avoid introducing zeros
-    fslmaths ${OutputT2wImage} -div ${T1wFolder}/BiasField_acpc_dc ${OutputT2wImage}_restore
-    fslmaths ${OutputT2wImage}_restore -mas ${T1wFolder}/${T1wImage}_acpc_dc_brain ${OutputT2wImage}_restore_brain
-  fi
-
+    if [ "$RunMode" -lt 5 ]; then
+        SpinEchoPhaseEncodeZero=""
+        case $AvgrdcSTRING in
+
+            ${FIELDMAP_METHOD_OPT} | ${SPIN_ECHO_METHOD_OPT} | ${GE_HEALTHCARE_LEGACY_METHOD_OPT} | ${GE_HEALTHCARE_METHOD_OPT} | ${SIEMENS_METHOD_OPT} | ${PHILIPS_METHOD_OPT})
+
+                log_Msg "Performing ${AvgrdcSTRING} Readout Distortion Correction"
+                wdir=${T2wFolder}/T2wToT1wDistortionCorrectAndReg
+                if [ -d ${wdir} ] ; then
+                    # DO NOT change the following line to "rm -r ${wdir}" because the
+                    # chances of something going wrong with that are much higher, and
+                    # rm -r always needs to be treated with the utmost caution
+                    rm -r ${T2wFolder}/T2wToT1wDistortionCorrectAndReg
+                fi
+
+                if [ $(imtest ${T2wFolder}/T2w) = 1 ] && [ "$UsePhaseZero" = "FALSE" && "$SPECIES" != "Human" ] ; then    # added T2w as a phase zero volume - TH Jan 2023
```

Comment:
fix logic (`UsePhaseZero` is never "FALSE", it is 0 or 1, and we want the true case, not the false case), reorder to make the "not human" test more obvious, use `[[ ]]` bash grammar instead of `[` builtin executable-like behavior.
```suggestion
                if [[ "$SPECIES" != "Human" && "$(imtest "$T2wFolder"/T2w)" == "1" && "$UsePhaseZero" == "1" ]] ; then    # added T2w as a phase zero volume - TH Jan 2023
```

Reply:
Since the option defaults to false anyway, do we actually want the "not human" test here?  Technically, it means that the option description should say something like "(disabled for human subjects)" to be accurate for the current logic.

Reply:
@glasserm thoughts on whether `--use-t2w-phase-zero=TRUE` should be nonfunctional if someone chooses to add it to a human run?  It defaults to false, so it matches current master behavior by default.

Reply:
That would be a new feature, and should be allowed.